### PR TITLE
HWKAPM-773 - Documented the usage of OpenShift env vars

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

File: docs/contributors/opentracingprovider.adoc [hunk 1]
```diff
@@ -177,6 +177,18 @@ The value should be used to set the componentType field on a `Component` node.
 
 If the transaction has not been defined in the trace context, then the value of this tag will be used to set it. This value will then be propagated to other subsequent trace contexts.
 
+==== Tag ‘service’
+
+A free-format value that ties this request to a specific service. When the application is executed in OpenShift, this defaults to a value that is derived from the environment variable `OPENSHIFT_BUILD_NAME`,
```
Comment (objectiser) [inline docs/contributors/opentracingprovider.adoc]: Shouldn't it also mention the HAWKULAR_APM_SERVICE_NAME as a container independent env variable, and what is precedence is to the OPENSHIFT... vars?

File: docs/contributors/opentracingprovider.adoc [hunk 2]
```diff
@@ -177,6 +177,18 @@ The value should be used to set the componentType field on a `Component` node.
...
+prefixed with the namespace specified via the environment variable `OPENSHIFT_BUILD_NAMESPACE`. For instance, if the `OPENSHIFT_BUILD_NAME` is `bar-1` and the `OPENSHIFT_BUILD_NAMESPACE` is `foo`, this
+value is `foo.bar`
```
Comment (objectiser) [inline docs/contributors/opentracingprovider.adoc]: Need to make it clear that the service name is independent of the specific version number?

File: docs/instrumentation/jvmagent.adoc [hunk 3]
```diff
@@ -11,7 +11,7 @@ This section will help you to configure a JVM based client environment to instru
 
 First step is to obtain the Hawkular APM distribution. If you have not already done so, then see the instructions on the link:SERVER.html[Server Installation] page.
 
-Once the Hawkular APM distribution zip has been unpacked, you will find a folder called *_apm_*. This folder contains the necessary environment script and agent jar(s) to configure a client JVM environment. 
+Once the Hawkular APM distribution zip has been unpacked, you will find a folder called *_apm_*. This folder contains the necessary environment script and agent jar(s) to configure a client JVM environment.
```
Comment (objectiser) [inline docs/instrumentation/jvmagent.adoc]: Not sure what changed here?

File: docs/contributors/opentracingprovider.adoc [hunk 4]
```diff
@@ -149,7 +149,7 @@ If the reference is to a local `Span` , then the nodeId of the referenced `Span`
 
 If the reference is based on extract trace state, then an `Interaction` based correlation id with the id provided in the trace state.
 
-NOTE: The ‘nodeId’ represents a composite of target fragment’s ‘fragmentId’ field and a sequence of indexes to locate the target node within the tree. For example, “abcd:0:2” represents a node in fragment ‘abcd’ which can be found by retrieving node 0 under the fragment, and then the third child node under that node. 
+NOTE: The ‘nodeId’ represents a composite of target fragment’s ‘fragmentId’ field and a sequence of indexes to locate the target node within the tree. For example, “abcd:0:2” represents a node in fragment ‘abcd’ which can be found by retrieving node 0 under the fragment, and then the third child node under that node.
```
Comment (objectiser) [inline docs/contributors/opentracingprovider.adoc:153]: Not sure what changed here?

File: docs/contributors/opentracingprovider.adoc (hunk 1 above)
Comment (objectiser) [reply docs/contributors/opentracingprovider.adoc]: Saw it is listed in the table below - but may be best to also describe here due to the dependency between the env vars if both set.

**Review by objectiser — CHANGES_REQUESTED** [review]

File: docs/instrumentation/jvmagent.adoc (hunk 3 above)
Comment (jpkrohling) [reply docs/instrumentation/jvmagent.adoc]: Trailing space, it seems. I blame Atom for that :)

File: docs/contributors/opentracingprovider.adoc (hunk 4 above)
Comment (jpkrohling) [reply docs/contributors/opentracingprovider.adoc:153]: Trailing space, I'll revert this.

**Review by objectiser — APPROVED** [review]

**Comment by objectiser** [comment]
@jpkrohling sorry for delay in re-reviewing - can you rebase as seem to be some conflicts, and then we can merge.

**Comment by jpkrohling** [comment]
Rebased.

**Comment by objectiser** [comment]
@jpkrohling Thanks!

**Pull request merged**